src/component/variantPage/FunctionalPrediction.tsx

```diff
@@ -63,32 +73,34 @@ class FunctionalPrediction extends React.Component<IFunctionalPredictionProps> {
     }
     public render() {
         const data = this.getData(this.props.variantAnnotation);
-        // Mutation Assessor only available in grch37
-        const shouldShowMutationAssessor =
-            SHOW_MUTATION_ASSESSOR &&
-            this.props.genomeBuild === GENOME_BUILD.GRCh37;
+        const shouldShowAlphaMissense = SHOW_ALPHAMISSENSE;
         return (
             <div>
                 <PolyPhen2
                     polyPhenScore={data.polyPhenScore}
                     polyPhenPrediction={data.polyPhenPrediction}
                 />
                 <Separator />
-                {shouldShowMutationAssessor && (
```

Comment:
Same here, we shouldn't remove the mutation assessor checking

Reply:
yes we should keep it, I was only referring to the `import SHOW_MUTATION_ASSESSOR` statement which seemed redundant to me but probably I overlooked those changes.

Reply:
changed back